django_pivot/pivot.py

```diff
@@ -1,30 +1,32 @@
-from django.db.models import Case, When, Q, F, Sum, CharField
-from django.db.models import Value
+import six
+from django.db.models import Case, When, Q, F, Sum, CharField, Value
 from django.shortcuts import _get_queryset
 
 from django_pivot.utils import get_column_values, get_field_choices
 
 
-def pivot(queryset, row, column, data, aggregation=Sum, choices='auto'):
+def pivot(queryset, rows, column, data, aggregation=Sum, choices='auto', prefix=''):
     """
     Takes a queryset and pivots it. The result is a table with one record
     per unique value in the `row` column, a column for each unique value in the `column` column
     and values in the table aggregated by the data column.
 
     :param queryset: a QuerySet, Model, or Manager
-    :param row: string, name of column that will key the rows
+    :param rows: list of strings, name of columns that will key the rows
     :param column: string, name of column that will define columns
     :param data: column name or Combinable
     :param aggregation: aggregation function to apply to data column
+    :param prefix: string added at a beginning data column name
     :return: ValuesQueryset
     """
+    values = [rows] if isinstance(rows, six.string_types) else list(rows)
+    row = values[0]
```

Comment:
Have you tried this when `get_field_choices` returns values on line 31?

Reply:
I'm going to write a test for this :)